Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivSession.KeyPairs.cs

```diff
@@ -129,6 +129,8 @@ public sealed partial class PivSession : IDisposable
                                             PivPinPolicy pinPolicy = PivPinPolicy.Default,
                                             PivTouchPolicy touchPolicy = PivTouchPolicy.Default)
         {
+            _yubiKeyDevice.ThrowIfUnsupportedAlgorithm(algorithm);
```

Comment:
This call might throw `NotSupportedException` which is not mentioned in the function's documentation above.